bin/src/command/orders.rs

```diff
@@ -1262,35 +1206,24 @@ impl CommandServer {
         for ref mut worker in self.workers.iter_mut().filter(|worker| {
             worker.run_state != RunState::Stopping && worker.run_state != RunState::Stopped
```

Comment:
Could you replace it with the `matches!` expression? Besides we should use the `.find` method instead of iterating on each worker and use the `if let Some` expression

Reply:
Not sure bout the `.find` method, as it finds only one item in a vector, but we want to send the requests to all workers